WooCommerce/WooCommerceTests/ViewRelated/Orders/Issue Refund/IssueRefundViewModelTests.swift

```diff
@@ -603,4 +614,32 @@ final class IssueRefundViewModelTests: XCTestCase {
         // Then
         XCTAssertTrue(chargeFetched)
     }
+
+    func test_viewModel_shows_spinner_when_charge_not_fetched_yet() {
```

Comment:
nice test cases! does it make sense to have a test case when the order doesn't have a charge ID?

Reply:
Yes, because then the spinner should never show. Added tests in 76a4646334f2b14644bbe4a910eeaccb4045e691, and also covered the empty string case.